src/hooks/use-fetch-wpcom-sites.tsx

```diff
@@ -99,6 +99,7 @@ export const useFetchWpComSites = ( connectedSites: SyncSite[] ) => {
 					fields: 'name,ID,URL,plan,is_wpcom_staging_site,is_wpcom_atomic,options',
 					filter: 'atomic,wpcom',
 					options: 'created_at,wpcom_staging_blog_ids',
+					site_visibility: 'visible',
```

Comment:
Why are we requesting the site visibility?, I don't think we are using it.

Reply:
Yep
I suppose Fredrik tried another approach and forgot to remove it

Reply:
This way we ignore deleted sites

Reply:
Yup, @nightnei is right 👍